### ISSUE TITLE
[Bug]: Getting Error in console

### ISSUE THREAD

**Issue by SideLo98** [Errored-Innovations/HeadsPlus#100]
### Checklist

- [X] Are you using the latest version/snapshot of [HeadsPlus](https://ci.pluginwiki.us/job/HeadsPlus/)?
- [X] Have you checked to see if others have submitted the same bug report?
- [X] Have you been able to reproduce the bug more then once?

### Description of the bug

https://pastebin.com/HRQ326jz

getting this error spam in console while killing a mob
it doesn't matter if mob is from Mythic Mob or Vanilla mob it gives error on both

### Steps To Reproduce

1. spawn a mob
2. kill it
3. gives error in console

### Additional Information

```shell
Server Version: Pufferfish version git-Pufferfish-53 (MC: 1.18.2) (Implementing API version 1.18.2-R0.1-SNAPSHOT) (Git: 79b9d86 on HEAD)
HeadsPlus Version: 7.0.4
MythicMobs Version: 5.0.1
```

**Comment by Thatsmusic99** [comment]
Quoting myself to one of my managers a couple of days ago, "I hate MythicMobs"

This dev did not hesitate to throw me overboard

**Comment by Thatsmusic99** [comment]
Fixed in v7.0.5 on SpigotMC
